src/relay/backend/te_compiler.cc

```diff
@@ -593,6 +593,14 @@ Pass LowerTensorExpr(TargetMap targets, DeviceMap device_context_map,
   return CreateFunctionPass(pass_func, 0, "LowerTensorExpr", {});
 }
 
+/*!
+ * \brief Obtain the Target from the device type.
+ * If homogenous compilation, this will return the only target.
+ * If heteregenous compilation, this will select associated using the targets_ Map.
+ *
+ * \param dev_type
+ * \return Target
+ */
```

Comment:
I'm all for more documentation but since this is declared and commented in the header the DNRY rule applies :-)

Reply:
Huh, I'm honestly not sure how this got in here-- maybe it was when I was moving stuff around.. I will remove and add @tkonolige's suggestion to the description in the header file